CHANGELOG.md

```diff
@@ -19,6 +19,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 2.0.0-beta.3 - 2022-XX-XX
+
+### Changed
+
+- Rename `StrongholdAdapterBuilder::build` to `StrongholdAdapterBuilder::try_build`;
```

Comment:
Isn't the rename from try_build to build?

Reply:
Good catch, it's fixed now, thanks :)